shared-module/displayio/Group.c

```diff
@@ -134,32 +134,28 @@ void displayio_group_construct(displayio_group_t* self, displayio_group_child_t*
     self->scale = scale;
 }
 
-bool displayio_group_get_pixel(displayio_group_t *self, int16_t x, int16_t y, uint16_t* pixel) {
-    x -= self->x;
-    y -= self->y;
-    // When we are scaled we need to substract all but one to ensure -scale to 0 divide down to -1.
-    // Normally -scale to scale both divide down to 0 because 0 is unsigned.
-    if (x < 0) {
-        x -= self->scale - 1;
-    }
-    if (y < 0) {
-        y -= self->scale - 1;
-    }
-    x /= self->scale;
-    y /= self->scale;
+bool displayio_group_get_area(displayio_group_t *self, displayio_buffer_transform_t* transform, displayio_area_t* area, uint32_t* mask, uint32_t* buffer) {
+    displayio_area_shift(area, -self->x * transform->scale, -self->y * transform->scale);
+    transform->scale *= self->scale;
+
+    bool full_coverage = false;
```

Comment:
I don't know what `full_coverage` means, but it's used several places, here and elsewhere, so an explanatory comment would be helpful where the term is used.

Reply:
I think it means there are no transparent pixels in that area.

Reply:
Yup, it's meant to track when we've filled in all pixels in an area. (We haven't if any of them are transparent.)

Reply:
Just a silly idea: if this was a count, instead of a bool, we would have an easy way to keep track of how many pixels are still left.